### PR TITLE
Quick cleanup

### PR DESCRIPTION
**Pull request by Thurman1776-**
Renaming things to introduce some alignment before it's too late.

### CODE REVIEW

**Comment by Thurman1776-** [comment]
@emrepun I'm also working on adding CI to this but at the moment it always fails 😅

File: Mac-App/Frontend/Frontend/Views/ExpandableList/DependencyItem.swift [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
 struct DependencyItem: View {
```
Comment (emrepun) [inline Mac-App/Frontend/Frontend/Views/ExpandableList/DependencyItem.swift]: I know this is not changed here, but do you think renaming `DependencyItem` to `DependencyItemView` would make sense?

**Review by emrepun — DISMISSED** [review]
🎉 Just left a minor comment.

File: Mac-App/Frontend/Frontend/Views/ExpandableList/DependencyItem.swift (hunk 1 above)
Comment (Thurman1776-) [reply Mac-App/Frontend/Frontend/Views/ExpandableList/DependencyItem.swift]: Ah nice catch! Yes, I'll do this here 👍

Comment (Thurman1776-) [reply Mac-App/Frontend/Frontend/Views/ExpandableList/DependencyItem.swift]: https://github.com/Thurman1776-/Uncomplect-us/pull/5/commits/aa98cf3619753885d52cc168664b92adad02bef5

**Review by emrepun — APPROVED** [review]

**Pull request merged**